### PR TITLE
docs(getting-started): Removed confusing infobox and corrected incorrect lifecycle name

### PR DESCRIPTION
**Pull request by Seebiscuit**
# Pull Request

## 📖 Description

I removed the Aurelia Architecture info box under Component Registration. I could not find a reference to the `StandardConfiguration` export mentioned there. It feels like that section is referencing an older part of the docs.

Also, did a best-effort attempt to update the lifecycle hook name based on the description.

### CODE REVIEW

**Comment by fkleuver** [comment]
The docs are in line with the latest `dev` version. I'm guessing you're working with `0.7.0`?

**Comment by Seebiscuit** [comment]
> The docs are in line with the latest `dev` version. I'm guessing you're working with `0.7.0`?

I'm not on any particular version, I was simply reading through the docs and found the difficult passage, @fkleuver.

File: docs/user-docs/getting-started/components.md [hunk 1]
```diff
@@ -278,7 +273,7 @@ Every lifecycle callback is optional. Implement whatever makes sense for your co
...
 | `detaching` | If your component has a method named "detaching", then the framework will invoke it when it is about to remove your HTML element from the document. In terms of the component hierarchy, the detaching hooks execute bottom-up. If you return a `Promise` (for example, from an outgoing animation), it will be awaited before the element is detached. It will run in parallel with promises returned from the `detaching` hooks of siblings / parents. |
-| `unbinding` | If your component has a method named "afterDetach", then the framework will invoke it when it has fully removed your HTML element from the document. In terms of the component hierarchy, the afterDetach hooks execute bottom-up. |
+| `afterDetach` | If your component has a method named "afterDetach", then the framework will invoke it when it has fully removed your HTML element from the document. In terms of the component hierarchy, the afterDetach hooks execute bottom-up. |
```
Comment (fkleuver) [inline docs/user-docs/getting-started/components.md:276]: Ah I see. So this actually needs to be removed. There is no `afterDetach` anymore.

**Comment by fkleuver** [comment]
Sorry for the lack of follow-up, we've been very busy reorganizing things and things have been updated in the meantime so I'll just close this.

**Pull request closed without merging**